stdlib/public/core/FloatingPointTypes.swift.gyb

```diff
@@ -53,7 +54,9 @@ elif Self == 'Double':
 
 elif Self == 'Float80':
     SelfDocComment = '''\
-/// An extended-precision, floating-point value type.'''
+/// An extended-precision, floating-point value type.
+///
+/// `Float80` is available on Intel, and unavailable on Apple silicon.'''
```

Comment:
This one is even more complex, it's probably best to say "Only available on Intel when the target system uses 80-bit long double" or similar.

Reply:
Changed in 4d35aae50e8.